chore(wit): Use versioned `wasi:logging` package

thomastaylor312 opened the pull request:
Publishing a package that depends on unversioned wit isn't allowed and it requires monkey patching when doing locally anyway. What this does mean is that all go examples need to use the versioned logging interface once we release the new wash. I'll be adding support for that in the host so it really only affects people who currently have Go components.

This is probably the easiest path forward for things unfortunately.

thomastaylor312 committed:
chore(wit): Use versioned `wasi:logging` package

Publishing a package that depends on unversioned wit isn't allowed and
it requires monkey patching when doing locally anyway. What this does
mean is that all go examples need to use the versioned logging interface
once we release the new wash. I'll be adding support for that in the host
so it really only affects people who currently have Go components.

This is probably the easiest path forward for things unfortunately.

Signed-off-by: Taylor Thomas <taylor@cosmonic.com>

lxfontes commented:
Can you please run `wit-deps` in all `_examples/` directories?
This will keep CI happy until we make them wkg compatible.

thomastaylor312 committed:
chore(examples): Updates all examples with new deps

We'll need to do a bit more cleanup (like removing the committed deps
folder) once wash releases

Signed-off-by: Taylor Thomas <taylor@cosmonic.com>
